src/spec-configuration/containerFeaturesOrder.ts

```diff
@@ -31,7 +31,7 @@ export function computeOverrideInstallationOrder(config: DevContainerConfig, fea
     // Moves to the beginning the features that are explicitly configured.
     const orderedFeatures = [];
     for (const featureId of config.overrideFeatureInstallOrder!) {
-        const feature = automaticOrder.find(feature => feature.features[0].name === featureId);
+        const feature = automaticOrder.find(feature => (feature.sourceInformation.type === 'oci' && feature.sourceInformation.referenceId.split(':')[0] === featureId) || feature.sourceInformation.referenceId === featureId);
```

Comment:
This might mean keeping track of two new variables in the `sourceInformation`.  That is nice since we can write static tests against the generated `featureConfig` before we actually use it anywhere (like here).

Reply:
Thanks for catching that. Modified condition and added test.